src/Umbraco.Infrastructure/Migrations/Expressions/Alter/Table/AlterTableBuilder.cs

```diff
@@ -21,7 +22,20 @@ public AlterTableBuilder(IMigrationContext context, AlterTableExpression express
 
     public ForeignKeyDefinition CurrentForeignKey { get; set; } = null!;
 
-    public void Do() => Expression.Execute();
+    public void Do()
+    {
+        if (_context.Database.DatabaseType.IsSqlite())
+        {
+            throw new NotSupportedException(@"SQLite do not support ALTER TABLE operations.Instead you will have to:
```

Comment:
A minor thing is a missing space before "Instead".
Could it use `Environment.NewLine` to list  each step on a separate line?

Reply:
Sure, we can use `Environment.NewLine`, but just a heads up, it will not be reflected into the error message on screen, but work in log messages.

Reply:
I guess it could handle replacements of newline here https://github.com/umbraco/Umbraco-CMS/blob/v10/contrib/src/Umbraco.Web.UI.Client/src/installer/steps/error.html#L4

Either a using a function, updating property value.

Reply:
It seems filters isn't injected into the install module. https://github.com/umbraco/Umbraco-CMS/blob/v10/contrib/src/Umbraco.Web.UI.Client/src/installer/_module.js

But alternatively it could probably be handled in the controller.